atomic_reactor/schemas/config.json

```diff
@@ -99,6 +99,10 @@
                         "additionalProperties": false
                     }
                 ]
+            },
+            "use_fast_upload": {
+                "description": "Use Koji's fast upload API",
```

Comment:
Do we want to update this text to say that this defaults to true?

Reply:
Done